StarRailBattleSim/src/relics/ornament/SpaceSealingStation.java

```diff
@@ -0,0 +1,36 @@
+package relics.ornament;
+
+import characters.AbstractCharacter;
+import powers.PermPower;
+import relics.AbstractRelicSetBonus;
+
+public class SpaceSealingStation extends AbstractRelicSetBonus {
+    public SpaceSealingStation(AbstractCharacter owner, boolean fullSet) {
+        super(owner, fullSet);
+    }
+
+    public SpaceSealingStation(AbstractCharacter owner) {
+        super(owner);
+    }
+
+    @Override
+    public void onEquip() {
+        this.owner.addPower(new SpaceSealingStationPower());
```

Comment:
the unconditional bonus should be a power in on equip, and the conditional bonus should be a power in `onCombatStart`

Reply:
Ah, do they not apply in combat screen even if you reach it there? I always assumed they did, the more you know. FIxed